docs/algorithms.md

```diff
@@ -125,7 +157,36 @@ To learn more about GradCAM, visit the following resources:
 - [Original paper](https://arxiv.org/abs/1610.02391)
 - [Website](http://gradcam.cloudcv.org/)
 
-Note: this method is currently available in master and will be available in pip / conda releases soon.
+## Layer Integrated Gradients
+Layer integrated gradients represents the integral of gradients with respect to the layer inputs / outputs along the straight-line path from the layer activations at the given baseline to the layer activation at the input.
```

Comment:
The layer and neuron ones look repetitive, since they are almost the same as the main ones. Perhaps at least referencing to the main one would be good ?